### PR TITLE
Rename `adaptive_scaffold` to `flutter_adaptive_scaffold`

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description

Nobody checked to see if `adaptive_scaffold` was already taken. It is, so we're renaming the package to `flutter_adaptive_scaffold` and publishing under that instead.

### CODE REVIEW

File: packages/flutter_adaptive_scaffold/example/pubspec.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ environment:
   flutter: ">=3.0.0"
 
 dependencies:
-  adaptive_scaffold:
-    path: ../../adaptive_scaffold
+  flutter_adaptive_scaffold:
+    path: ../../flutter_adaptive_scaffold
```
Comment (stuartmorgan) [inline packages/flutter_adaptive_scaffold/example/pubspec.yaml:12]: Optional nit: This could just be `path ../`

**Review by stuartmorgan — APPROVED** [review]
LGTM once `format` is happy.

**Review by HansMuller — APPROVED** [review]
LGTM

File: packages/flutter_adaptive_scaffold/pubspec.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-name: adaptive_scaffold
+name: flutter_adaptive_scaffold
 description: Widgets to easily build adaptive layouts, including navigation elements.
 version: 0.0.3
-issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+adaptive_scaffold%22
-repository: https://github.com/flutter/packages/tree/main/packages/adaptive_scaffold
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+flutter_adaptive_scaffold%22
```
Comment (nilsreichardt) [inline packages/flutter_adaptive_scaffold/pubspec.yaml:4]: The label "flutter_adaptive_scaffold" does exit in the Flutter repository. I created a ticket to rename the "adaptive_scaffold" label to "flutter_adaptive_scaffold": https://github.com/flutter/flutter/issues/111832

**Pull request merged**